features/security/src/main/feature/feature.xml

```diff
@@ -142,6 +142,14 @@
         <bundle>mvn:ddf.security.handler/security-handler-api/${project.version}</bundle>
     </feature>
 
+    <feature name="security-migratable" version="${project.version}"
+             description="Exports security configurations and system files">
+        <feature>guava</feature>
+        <bundle>mvn:ddf.platform.migration/platform-migratable-api/${project.version}</bundle>
+        <bundle>mvn:ddf.platform.api/platform-api/${project.version}</bundle>
```

Comment:
❓ Is there a reason you didn't just add the `<feature>platform-migratable-api</feature>` dependency which starts both of the above bundles?

Reply:
circular dependencies